### ISSUE TITLE
Grey line on top of the browser.

### ISSUE THREAD

**Issue by berezki** [zen-browser/desktop#1737]
### Captchas

- [x] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

![изображение](https://github.com/user-attachments/assets/30bb4252-7a55-420e-8645-35cd6e3d5685)
There is a grey line one pixel width at the top of the browser window.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.5

### What platform are you seeing the problem on?

Linux

**Comment by coderstarrer** [comment]
It's because of a theme, did you install a theme called Zen Paper Cut Theme because it makes it like that

**Comment by berezki** [comment]
> It's because of a theme
Yes, thank you, but It's because of the default "System Theme - Auto theme". Switched to dark and everything is fine now. But still looks like an Zen Browser issue needed to be fixed (as this is one of the default themes and there is clearly shouldn't be any stripes and lines).

**Comment by n7itro** [comment]
Can't reproduce

**Comment by kpuljek** [comment]
Came here looking for this bug. Same issue, grey line when using the System Theme - Auto theme. 
![image](https://github.com/user-attachments/assets/07632e97-d901-4ae8-953a-5a8ffe08586b)

It's not there when I switch to any other theme. (EDIT: Not entirely true, most themes are fine, but some also have the line)

**Comment by kpuljek** [comment]
The line is still there. It starts where the sidebar ends (above the "back button") and continues to the end of the window, but it no longer goes outside the window so it's much less jarring.

Some examples:

System auto - system set to dark
![image](https://github.com/user-attachments/assets/24538d9e-b964-4672-a0ae-11feb3817722)

Dark
![image](https://github.com/user-attachments/assets/7e0abe70-90ba-4dff-9390-adee4b76b3de)

System auto - system set to light
![image](https://github.com/user-attachments/assets/569f4834-796e-473d-8651-bea234816436)

Light
![image](https://github.com/user-attachments/assets/b2550e84-466d-41fa-af25-d524277c488d)

**Comment by salamijason** [comment]
this was tested and recreated when having the system-auto theme enabled. right? @neurokitti

**Comment by neurokitti** [comment]
> this was tested and recreated when having the system-auto theme enabled. right? @neurokitti

yup

**Comment by JSGRD22** [comment]
Has this been resolved

**Comment by salamijason** [comment]
no response in 2 weeks, closing.
